runtime/src/accounts_db.rs

```diff
@@ -724,7 +777,9 @@ impl AccountsDB {
         let mut infos: Vec<AccountInfo> = vec![];
         while infos.len() < with_meta.len() {
             let storage = self.find_storage_candidate(fork_id);
-            let rvs = storage.accounts.append_accounts(&with_meta[infos.len()..]);
+            let rvs = storage
+                .accounts
+                .append_accounts(&with_meta[infos.len()..], &hashes);
```

Comment:
This is not correct; see here: https://github.com/solana-labs/solana/pull/5573/files#diff-2099c5256db4eb5975c8834af38f6456R782